MyWeightTests/SnapshotService.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SnapshotService.swift
+//  MyWeight
+//
+//  Created by Bruno Mazzo on 11/5/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import Foundation
+import Nimble
+import Nimble_Snapshots
+
+public struct SnapshotService {
+    let iPhonePortraitSizes = ["iPhone 4'" : CGSize(width: 320, height: 568),
+                       "iPhone 4.7'" : CGSize(width: 375, height: 667),
+                       "iPhone 5.5'" : CGSize(width: 414, height: 736)]
+
+    public struct ScreenSnapshot {
```

Comment:
This is great! I only would drop the `Screen` from the struct name (and on the function names), what do you think about just `Snapshot`?